man pages installation does not work unless `man/man1` directory already exists

## Version of Singularity:

master

Note that the man pages cannot be installed to a custom prefix if the directory `man/man1` does not already exist.

```
[vagrant@localhost singularity]$ sudo make -C builddir man
make: Entering directory `/home/vagrant/singularity/builddir'
FATAL   [U=0,P=31350]      assertAccess()                Given directory (/usr/local/singularity/share/man/man1) does not exist or is not writable by
 calling user.
exit status 255
make: *** [man] Error 1
make: Leaving directory `/home/vagrant/singularity/builddir'
```

--- a/cmd/docs/docs.go
+++ b/cmd/docs/docs.go
@@ -17,7 +17,7 @@ import (
 
 func assertAccess(dir string) {
 	if err := unix.Access(dir, unix.W_OK); err != nil {
-		sylog.Fatalf("Given directory does not exist or is not writable by calling user.")
+		sylog.Fatalf("Given directory (%s) does not exist or is not writable by calling user", dir)
 	}
 }
 

--- a/mlocal/frags/Makefile.stub
+++ b/mlocal/frags/Makefile.stub
@@ -2,6 +2,7 @@ all: $(ALL)
 
 .PHONY: man
 man:
+	mkdir -p $(DESTDIR)$(MANDIR)/man1
 	$(V)$(GO) run $(GO_MODFLAGS) -tags "$(GO_TAGS)" $(GO_GCFLAGS) $(GO_ASMFLAGS) \
 		$(SOURCEDIR)/cmd/docs/docs.go man --dir $(DESTDIR)$(MANDIR)/man1
 

